gateway/src/main/resources/application.yml

```diff
@@ -70,6 +70,7 @@ georchestra:
   gateway:
     security:
       create-non-existing-users-in-l-d-a-p: false
+      createDefaultOrg: ${createDefaultOrg:false}
```

Comment:
I would have expected to have the `createDefaultOrg` boolean property onto the previous class and having a `if` statement making use of it, but actually you can have `createDefaultOrg` set to false, as long as org is empty and a defaultOrganization string is defined, then we associate the user to the default organization, but I am confused about what is the intent behind the PR.